examples/deployment/README.md

```diff
@@ -0,0 +1,731 @@
+       -DpomFile=$COHERENCE_HOME/plugins/maven/com/oracle/coherence/coherence/12.2.1/coherence.12.2.1.pom
+  
+   mvn install:install-file -Dfile=$COHERENCE_HOME/lib/coherence-metrics.jar \
+       -DpomFile=$COHERENCE_HOME/plugins/maven/com/oracle/coherence/coherence-metrics/12.2.1/coherence-metrics.12.2.1.pom
+   ```
+
+## Create the example namespace
+
+You need to create the namespace for the first time to run any of the examples. Create your target namespace:
+
+```bash
+kubectl create namespace coherence-example-ns
+
+namespace/coherence-example-ns created
+```
+
+> Note: In the examples, a Kubernetes namespace called `coherence-example-ns` is used. 
+> If you want to change this namespace, ensure that you change any references to this namespace 
+> to match your selected namespace when running the examples.
+
+## Get Coherence Docker image
+
+Get the Coherence Docker image from the Oracle Container Registry:
+
+1. In a web browser, navigate to [Oracle Container Registry](https://container-registry.oracle.com) and click Sign In.
+1. Enter your Oracle credentials or create an account if you don't have one.
+1. Search for `coherence` in the Search Oracle Container Registry field.
+1. Click coherence in the search result list.
+1. On the Oracle Coherence page, select the language from the drop-down list and click Continue.
+1. Click `Accept` on the Oracle Standard Terms and Conditions page.
+
+> Note: You may pull this image and place in your own repository or pull directly from Oracle Container Registry.
+
+## Create a secret
+
+If all of your images can be pulled from public repositories, this step is not
+required. Otherwise, you need to enable your Kubernetes cluster to pull
+images from private repositories. You must create a secret to convey
+the docker credentials to Kubernetes. In the examples,
+the secret named `coherence-example-secret` is used in the namespace `coherence-example-ns`.
+
+```bash
+kubectl -n coherence-example-ns \
+  create secret docker-registry coherence-example-secret \
+  --docker-server=$DOCKER_REPO \                              <1>
+  --docker-username=$DOCKER_USERNAME \                        <2>
+  --docker-password=$DOCKER_PASSWORD \                        <3>
+  --docker-email=$DOCKER_EMAIL                                <4>
+```
+
+* Replace `$DOCKER_REPO` with the name of the Docker repository that the images are to be pulled from.
+* Replace `$DOCKER_USERNAME` with your username for that repository.
+* Replace `$DOCKER_PASSWORD` with your password for that repository.
+* Replace `$DOCKER_EMAIL` with your email (or even a fake email).
+
+See the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/)
+on pull secrets for more details.
+
+## Clone the GitHub repository
+
+The examples exist in the `2.0` branch of the Coherence Operator GitHub repository - https://github.com/oracle/coherence-operator.
```

Comment:
You will need to change this when we merge `2.0` onto `master` as once we release we will blow away all the old 1.0 code and 2.0 will be the only source.

Reply:
ok understand.